src/App.jsx

```diff
@@ -1,10 +1,76 @@
-import React from 'react';
+import React, { Component } from "react";
+import axios from "axios";
 
-function App() {
-  return (
-    <div>
-    </div>
-  );
+class App extends Component {
+  state = {
+    query: "",
+    results: [],
+    message: ""
+  };
+
+  onSubmitFormHandler = async e => {
+    e.preventDefault();
+    let response = await axios.post("https://cocktails-api-team1.herokuapp.com/api/v1/cocktails", {
+      params: {
+        query: "name"
```

Comment:
Why are you sending of a hard-coded value in that string?

Reply:
We are currently looking into this issue.

Reply:
This has been updated to send in q: e.target.query.value instead of the hard coded value of "name"